users/service.go

```diff
@@ -517,7 +513,7 @@ func (svc service) retrieveObjectUsersPermissions(ctx context.Context, domainID,
 	userID := auth.EncodeDomainUserID(domainID, client.ID)
 	permissions, err := svc.listObjectUserPermission(ctx, userID, objectType, objectID)
 	if err != nil {
-		return err
+		return errors.Wrap(svcerr.ErrFailedPermissionsList, err)
```

Comment:
From user prespective , this can be a internal error.
@WashingtonKK What do you think? Please give your comments

Reply:
Yes, it needs to be an internal error but we still need to notify the user of an error.

We can change it to an errAuthorization instead.
What do you think about this @arvindh123 ?

Reply:
I see  `svc.listObjectUserPermission` already wrap with error `svcerr.ErrFailedPermissionsList`, we can make `svcerr.ErrFailedPermissionsList` as internal error and return `svcerr.ErrAuthroization` or something like denied

Reply:
This make sense, okay